dll/shellext/mydocs/CMyDocsDropHandler.cpp

```diff
@@ -157,7 +157,9 @@ CMyDocsDropHandler::Drop(IDataObject *pDataObject, DWORD dwKeyState,
     fileop.pFrom = pszzSrcList;
     fileop.pTo = szzDir;
     fileop.fFlags = FOF_ALLOWUNDO | FOF_FILESONLY | FOF_MULTIDESTFILES | FOF_NOCONFIRMMKDIR;
-    SHFileOperationW(&fileop);
+    hr = SHFileOperationW(&fileop);
+    if (hr)
+        ERR("SHFileOperationW failed with 0x%x\n", GetLastError());
```

Comment:
msdn states:
> Do not use GetLastError with the return values of this function.

Reply:
Good point, I did not notice... but we already do it elsewhere in ROS for this function... i'll fix !